payment/controllers.go

```diff
@@ -133,17 +240,12 @@ func GetTransactions(service *Service) handlers.AppHandler {
 
 		id := uuid.Must(uuid.FromString(orderID))
 
-		order, err := service.datastore.GetTransactions(id)
+		transactions, err := service.datastore.GetTransactions(id)
 		if err != nil {
-			return handlers.WrapError(err, "Error retrieving the transactions for the order", http.StatusInternalServerError)
+			return handlers.WrapError(err, "Error retrieving the transactions for the transactions", http.StatusInternalServerError)
```

Comment:
can remove "for the transactions"

Reply:
https://github.com/brave-intl/bat-go/pull/285/commits/a6d0759ecda6f4af8a591b31b6acd5619d18a131